plugin/src/test/java/org/opensearch/ml/rest/RestMLRemoteInferenceIT.java

```diff
@@ -457,6 +428,7 @@ public void testOpenAIEditsModel() throws IOException, InterruptedException {
         assertFalse(((String) responseMap.get("content")).isEmpty());
     }
 
+    @Ignore
```

Comment:
Any reason for ignoring these tests?

Reply:
Running the text-embedding model should be enough for OpenAI. No need to verify every other models to reduce the IT time. This moderations model is not quite used anyways for now.